.github/workflows/publish-release-reusable.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish release (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      VERSION:
+        required: true
+        type: string
+        description: The semantic version number.
+    secrets:
+      BOT_PAT:
+        required: true
+        description: The github personal access token of your bot.
+
+jobs:
+  publish-release:
+    name: Publish release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Publish
+        env:
+          VERSION: ${{ inputs.VERSION }}
+          GH_TOKEN: ${{ secrets.BOT_PAT }}
+        shell: bash
+        run: |
+          gh release edit "${VERSION}" --draft=false --latest
```

Comment:
you have `--latest` flag because if set the latest release is edited. We want to edit the VERSION. Should we delete the latest flag?